### PR TITLE
Add dummy data

### PR DESCRIPTION
**Pull request by jelneo**

### CODE REVIEW

File: db/fillTables.js [hunk 1]
```diff
@@ -28,11 +28,45 @@ const Staffs = [
     [12, 'Pom', 'iamvegan', 'password22', 'iambatman', 2],
 ];
 
+// (rid, minSpending (in cents), rname)
 const Restaurants = [
     [1, 5, 'Fukuroku'],
     [2, 10, 'MaMas Specials']
```
Comment (MuhdNurKamal) [inline db/fillTables.js]: don't forget the commas

File: db/fillTables.js [hunk 2]
```diff
@@ -28,11 +28,45 @@ const Staffs = [
...
-    [2, 10, 'MaMas Specials']
+    [2, 10, 'MaMas Specials'],
+    [3, 11, 'MacDonalds'],
+    [4, 15, 'KFC']
 ];
 
+// (pid, points, startDate, endDate, percentOff, minSpending (in cents), monthsWithNoOrders)
+const GlobalPromotions = [
+    // 10% off all orders for one month with min spend $30
+    [1, NULL, '2019-06-01', '2019-07-01', 10, 3000, NULL],
```
Comment (MuhdNurKamal) [inline db/fillTables.js]: NULL is not a js type. need use 'NULL'

**Comment by whenzei** [comment]
LGTM except for what @MuhdNurKamal raised

**Comment by MuhdNurKamal** [comment]
LGTM

**Pull request merged**